doc/image-from-registry.md

```diff
@@ -0,0 +1,42 @@
+# Creating a registry image with a VM disk
+The purpose of this document is to show how to create registry image containing a Virtual Machine image that can be import into a PV.
+
+## Prerequisites
+The VM disk image should be called 'disk.img' and placed in the root folder.
+
+## Create an image with Buildah
+
+Create a new container from scratch and keep its name in a variable:
+
+```bash
+$ newcontainer=$(buildah from scratch)
+```
+
+Add the VM disk to the image and commit:
+
+```bash
+$ buildah copy $newcontainer ./disk.img /
+buildah commit $newcontainer my-vm-img
+```
+
+To push an image from your local Buildah container storage, check the image name, then push it it using the buildah push command. Remember to identify both the local image name and a new name that includes the location (<registry>:5000, in this case):
+```
+buildah push --tls-verify=false my-vm-img:latest <registry>:5000/my-vm-img:latest
+```
+
+## Create an image with Docker
+
+Create a Dockerfile with the following content:
+
+```
+FROM scratch
+ADD disk.img /
+```
+
+Build, tag and push the image:
+
+```bash
+$ docker build . -t my-vm-img
+$ docker tag my-vm-img <registry>:5000/my-vm-img
```

Comment:
Don't you want to tag it with 'latest' as well?

Reply:
It will get it automatically